src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    private VerifyData verifyData = new VerifyData();
```

Comment:
Ensure that the `VerifyData` class is implemented correctly and that the `verification` method checks the data format as expected.